src/css/App.css

```
@@ -60,4 +60,14 @@ body {
 
 .speed {
   color: #fff;
+}
+
+.appName{
+  text-align: center;
+  font-size: 5rem;
+}
+
+.gitBtn{
+  margin-left: 1rem;
+  margin-top: 1rem;
```

```suggestion
  text-align: right;
  padding: 15px;
```

This will make it float to the right